pkg/api-server/server.go

```diff
@@ -71,7 +73,8 @@ func init() {
 	}
 }
 
-func NewApiServer(resManager manager.ResourceManager, defs []definitions.ResourceWsDefinition, cfg *kuma_cp.Config, enableGUI bool, metrics metrics.Metrics) (*ApiServer, error) {
+func NewApiServer(resManager manager.ResourceManager, wsManager customization.APIInstaller,
+	defs []definitions.ResourceWsDefinition, cfg *kuma_cp.Config, enableGUI bool, metrics metrics.Metrics) (*ApiServer, error) {
```

Comment:
formatting nit. For me, it's easier to read arguments grouped in one line or every in it's own line so either
```
func NewApiServer(resManager manager.ResourceManager, wsManager customization.APIInstaller, defs []definitions.ResourceWsDefinition, cfg *kuma_cp.Config, enableGUI bool, metrics metrics.Metrics) (*ApiServer, error) {
```
or
```
func NewApiServer(
    resManager manager.ResourceManager,
    wsManager customization.APIInstaller,
    defs []definitions.ResourceWsDefinition,
    cfg *kuma_cp.Config,
    enableGUI bool,
    metrics metrics.Metrics,
) (*ApiServer, error) {
```